todo_app/app.py

```diff
@@ -1,28 +1,30 @@
-from flask import Flask, render_template, request
+from flask import Flask, render_template, request, redirect
 from todo_app.flask_config import Config
 from todo_app.data.session_items import *
+from todo_app import item
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-# I added add functionality to the route rather than a seperate page as this seemed more efficient
-@app.route('/', methods=['POST', 'GET'])
+# Populate lists and add items 
+@app.route('/',methods=['POST', 'GET'])
```

Comment:
I think the comment is out of date now you have a separate `/add` endpoint, and you don't need to allow POST requests